img/carrot.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="16px" height="10px" viewBox="0 0 16 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <defs></defs>
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M7.89666325,0.908974757 L7.90897476,0.896663247 L15.3966632,8.38435174 L13.8505329,9.9304821 L7.89666325,3.97661247 L1.94279361,9.9304821 L0.396663247,8.38435174 L7.88435174,0.896663247 L7.89666325,0.908974757 Z" fill="#FFFFFF"></path>
+    </g>
+</svg>
```

Comment:
I think you meant, "caret." 

Also, SVG files like this should be run through an optimization tool such as https://jakearchibald.github.io/svgomg/; I see a number of attributes that could be stripped out to save weight.